pkg/kyma/cmd/test/cmd.go

```diff
@@ -0,0 +1,155 @@
+package test
```

Comment:
why is that a whole new package, not just a new test.go in package above?

Reply:
I just followed the convention. `install` and `uninstall` have their own packages. Now I see that they also have subcommands, that's why they are separated. Should I flatten `test` package?

Reply:
I've flattened the directory